Packages/ohif-wadoproxy/server/routes.js

```diff
@@ -27,146 +27,153 @@ const authenticateUser = request => {
     });
 };
 
+const handleRequest = function() {
```

Comment:
Can you made the wadoproxy changes a separate PR? They are small and self contained.